cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -471,23 +471,23 @@ function LogViewerController ($scope, LogViewerStore, myLogsApi, LOGVIEWERSTORE_
 
     this.rawUrl = url;
 
-    myLogsApi.getLogsStartAsJson({
+    myLogsApi.nextLogsJsonOffset({
         namespace : this.namespaceId,
         appId : this.appId,
         programType : this.programType,
         programId : this.programId,
         runId : this.runId,
-        start : this.startTimeSec
+        fromOffset: -1 + '.' + this.startTimeSec*1000
```

Comment:
It is -1 when we change start time and we need to produce a fromOffset value rather than fetching it from the last item in the response.